sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -345,17 +343,19 @@ class ScrollableBlock extends Component {
 }
 
 /// An optimized scrollable widget for a large number of children that are all
-/// of the same height. Use this widget when you have a large number of children
-/// or when you are concerned about offscreen widgets consuming resources.
+/// the same size in the scrollDirection. For example for ScrollDirection.vertical
+/// itemSize is the height of each item. Use this widget when you have a large
+/// number of children or when you are concerned about offscreen widgets
+/// consuming resources.
 abstract class FixedHeightScrollable extends Scrollable {
```

Comment:
This class seems incorrectly named now...  Maybe FixedSizeScrollable because all the items really do have the same size (in both height and width)?  We can rename it in a future patch if you prefer.